### DISCUSSION TITLE
Please Approval of WalletConnect registry

### DISCUSSION THREAD

**Discussion by SnailMyth** [WalletConnect/walletconnect-monorepo#1089] [Developer Support]
Hi there!
We have submitted an application to register our wallet "Hideout Wallet" for somedays.
Would you be so kind as to approve the application so that our users can access WallectConnect as soon as possible.
Thanks in advance.

**Comment by hackyguru** [comment]
Hello. thanks for submitting to our registry. I'll keep you updated about the status of your wallet after the review in this thread.
